ui/src/containers/BasicDetails/index.tsx

```diff
@@ -163,12 +163,20 @@ const BasicDetails: React.FC = () => {
             <hr />
             <div>
               <TextContent>
-                <Text component={TextVariants.h2}>Install on Kubernetes</Text>
+                <Text component={TextVariants.h2}>Install using kubectl</Text>
                 <Text> {resource.kind.name} </Text>
                 <ClipboardCopy isReadOnly variant={ClipboardCopyVariant.expansion}>
                   {resource.installCommand}
                 </ClipboardCopy>
               </TextContent>
+              <br />
+              <TextContent>
```

Comment:
I think we can shift this part to components and reuse this as we have used it twice ?